### ISSUE TITLE
Problem with mamba/conda install

### ISSUE THREAD

**Issue by koolape** [rrwick/Trycycler#13]
**I got the following error when I tried to install Trycycler with mamba (another anaconda package mangaer):**

`I got the following error when I tried to install Trycycler with mamba (another anaconda package mangaer):

Looking for: ['trycycler']

bioconda/linux-64        [====================] (00m:00s) No change
bioconda/noarch          [====================] (00m:00s) No change
anaconda/pkgs/main/linux [====================] (00m:00s) No change
anaconda/pkgs/r/linux-64 [====================] (00m:00s) No change
anaconda/pkgs/main/noarc [====================] (00m:00s) No change
anaconda/pkgs/r/noarch   [====================] (00m:00s) No change
anaconda/pkgs/msys2/linu [====================] (00m:00s) No change
anaconda/pkgs/msys2/noar [====================] (00m:00s) No change
Encountered problems while solving.
Problem: nothing provides icu 54.* needed by r-base-3.4.1-0`

**And I got the following error when I tried to install Trycycler with conda:**
`Found conflicts! Looking for incompatible packages.                                      failed

UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions`

**Comment by rrwick** [comment]
I'm no expert with conda stuff, but I've done some experimenting and I may have found a solution.

I managed to reproduce your issue (or at least something very similar) on Ubuntu 18.04. I narrowed the problem to the `r-phangorn` package, which is one of the dependencies in the [Trycycler conda recipe](https://github.com/bioconda/bioconda-recipes/blob/master/recipes/trycycler/meta.yaml). I then found [this issue](https://github.com/conda/conda/issues/7646) which highlighted the problem: I didn't have the `conda-forge` channel set up, which is a requirement for Bioconda.

Once I fixed that (`conda config --add channels conda-forge`) I was able to successfully run `conda install trycycler`. And I've taken what I've learned in this process and updated the [Software requirements](https://github.com/rrwick/Trycycler/wiki/Software-requirements) and [Installation](https://github.com/rrwick/Trycycler/wiki/Installation) pages of the wiki.

I hope that fixes the issue for you too! If not, let me know.

Ryan
